javascript/javascript-import-and-require.md

```diff
@@ -0,0 +1,79 @@
+# JavaScript - Require & Import
+
+## Project Time
+
+About 45 Minutes
+
+- 30 minutes for the video and reference material
+- 10 minutes for Check for Understanding
+- 5 minutes for verify
+
+## Prerequisites
+
+- JavaScript 7
+- [JavaScript Import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)
+- [JavaScript Require](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+
+## Motivation
+
+JavaScript is one of most used language. It is lightweight, Interpreted and Just-in-time compiled programming language with first-class function. While it is most well-known as the scripting language for webpage, Many no-browser environmemts also use it. JavaScript is a prototype-based, multi-paradigm, single-threaded, dynamic language, supporting object-oriented, imperative and declarative style.
+
+## Objectives
+
+- Create understanding in user regarding of import and require
+
+## Supplemental Materials
+
+- [JavaScript reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference)
+- [JavaScript Modules: ES6 Import and Export](https://www.youtube.com/watch?v=_3oSWwapPKQ)
+- [Javascript require vs import in 3 min](https://www.youtube.com/watch?v=vGEnYyO_r2g)
+- [Node JS Tutorial: Modules and require()](https://www.youtube.com/results?search_query=understand+require+in+javascript)
+
+## Lesson
+
+**Why both used required in today?**

```

Comment:
I don't understand what this statement means.

Reply:
It is a mistake, I will correct it .